apps/yggdrasil/priv/repo/seeds.exs

```diff
@@ -9,3 +9,78 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+require Logger
+
+alias Yggdrasil.{Repo, Resource, Permission, Role, RolePermission}
+
+perms = ["read", "write", "all"]
+res = ["character", "game"]
+
+Repo.transaction fn ->
+  Enum.each perms, fn p ->
+    if Repo.get(Permission, p) == nil do
+      Repo.insert! %Permission{name: p}
+    else
+      Logger.warn fn -> "found permission #{p} skipping insert" end
```

Comment:
Wanted it to stand out in the build logs. Locally we expect to see it frequently but in the ci build logs I sort of wanted it to jump out. If they exist great, but would like to know why.